### ISSUE TITLE
Lab 2 Ex 2 Task 3 Steps 2 and 3 "Blobs" has become "Containers"

### ISSUE THREAD

**Issue by davidcwalls** [MicrosoftLearning/DP-200-Implementing-an-Azure-Data-Solution#31]
Currently, the steps are:
2. In the **awsastudxx** screen, where **xx** are your initials, under the **Services** click **Blobs**.
3. In the **awsastudxx - Blobs** screen, at the top left, click on the **+ Container** button.

It appears that **Blobs** has been renamed **Containers** in the Azure UI (see screenshot below)

I suggest rewording the steps along the lines of:
2. In the **awsastudxx** screen, where **xx** are your initials, in the **Services** section click **Containers**.
3. In the **awsastudxx** - **Containers** screen, at the top left, click on the **+ Container** button.

![Storage Account Containers](https://user-images.githubusercontent.com/32783995/67209530-8a02a400-f3e5-11e9-8108-30c693be3b2c.png)
![Storage Account Containers 2](https://user-images.githubusercontent.com/32783995/67210567-4dd04300-f3e7-11e9-9eb3-b425c9b8e092.png)

**Comment by ctesta-oneillmsft** [comment]
resolved with graphics
